spec/services/dialog_local_service_spec.rb

```diff
@@ -168,13 +174,27 @@
                        "host", "hosts", "/host"
     end
 
+    context "when the object is a HostEsx" do
+      let(:obj) { double(:class => ManageIQ::Providers::Vmware::InfraManager::HostEsx, :id => 123) }
+
+      include_examples "DialogLocalService#determine_dialog_locals_for_custom_button return value",
+                       "host", "hosts", "/host"
+    end
+
     context "when the object is an InfraManager" do
       let(:obj) { double(:class => ManageIQ::Providers::Vmware::InfraManager, :id => 123) }
 
       include_examples "DialogLocalService#determine_dialog_locals_for_custom_button return value",
                        "ext_management_system", "providers", "/ems_infra"
     end
 
+    context "when the object is a CloudManager" do
+      let(:obj) { double(:class => ManageIQ::Providers::Vmware::CloudManager, :id => 123) }
+
+      include_examples "DialogLocalService#determine_dialog_locals_for_custom_button return value",
+                       "ext_management_system", "providers", "/ems_cloud"
```

Comment:
Is "/ems_cloud" right?  Do you know how that is used?

Reply:
Oh, is that a UI route?

Reply:
Yeah but it'd be real spiffing if someone on the UI team could confirm that that's what is supposed to happen here.